src/transformers/models/esm/modeling_esmfold.py

```diff
@@ -2100,7 +2100,7 @@ def forward(
 
         >>> model = EsmForProteinFolding.from_pretrained("facebook/esmfold_v1")
         >>> tokenizer = AutoTokenizer.from_pretrained("facebook/esmfold_v1")
-        >>> inputs = tokenizer(["MLKNVQVQLV"], return_tensors="pt")  # A tiny random peptide
+        >>> inputs = tokenizer(["MLKNVQVQLV"], return_tensors="pt", add_special_tokens=False)  # A tiny random peptide
```

Comment:
Trying to run the example - this fix gets us one step further :) but example still errors out for me
```
Traceback (most recent call last):
  File "/private/home/tsercu/work/112/transformers/docs/source/en/model_doc/examle.py", line 6, in <module>
    outputs = model(**inputs)
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/transformers/models/esm/modeling_esmfold.py", line 2134, in forward
    esm_s = self.compute_language_model_representations(esmaa)
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/transformers/models/esm/modeling_esmfold.py", line 2231, in compute_language_model_representations
    esm_hidden_states = self.esm(esmaa, attention_mask=esmaa != 1, output_hidden_states=True)["hidden_states"]
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/transformers/models/esm/modeling_esm.py", line 869, in forward
    encoder_outputs = self.encoder(
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/transformers/models/esm/modeling_esm.py", line 572, in forward
    layer_outputs = layer_module(
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/transformers/models/esm/modeling_esm.py", line 457, in forward
    self_attention_outputs = self.attention(
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/transformers/models/esm/modeling_esm.py", line 390, in forward
    hidden_states_ln = self.LayerNorm(hidden_states)
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/torch/nn/modules/normalization.py", line 190, in forward
    return F.layer_norm(
  File "/private/home/tsercu/.conda/envs/esm39/lib/python3.9/site-packages/torch/nn/functional.py", line 2515, in layer_norm
    return torch.layer_norm(input, normalized_shape, weight, bias, eps, torch.backends.cudnn.enabled)
RuntimeError: "LayerNormKernelImpl" not implemented for 'Half'
```

Reply:
That's because of the default to use the ESM model in half precision [here](https://huggingface.co/facebook/esmfold_v1/blob/main/config.json#L15), half is not implemented on CPU. @Rocketknight1 we should change the config to set it to `False`.

Reply:
Done!